src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/Common/NameValuePair.cs

```diff
@@ -1,71 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-namespace Microsoft.Data.Common
-{
-
-    using System;
-    using System.Diagnostics;
-    using System.Runtime.Serialization;
-
-    [Serializable] // MDAC 83147
-    sealed internal class NameValuePair
-    {
-        readonly private string _name;
-        readonly private string _value;
-        [OptionalField(VersionAdded = 2)]
-        readonly private int _length;
-        private NameValuePair _next;
-
-        internal NameValuePair(string name, string value, int length)
-        {
-            System.Diagnostics.Debug.Assert(!ADP.IsEmpty(name), "empty keyname");
-            _name = name;
-            _value = value;
-            _length = length;
-        }
-
-        internal int Length
-        {
-            get
-            {
-                // this property won't exist when deserialized from Everett to Whidbey
-                // it shouldn't matter for DbConnectionString/DbDataPermission
-                // which should only use Length during construction
-                // not deserialization or post-ctor runtime
```

Comment:
Why not bring this comment block over to the new file?

Reply:
Everett to Whidbey are .net 1.1 and 2.0, neither is supported so the comment is no longer relevant.